Kickstarter-iOS/Views/Controllers/SelectCurrencyViewController.swift

```diff
@@ -36,11 +36,16 @@ final class SelectCurrencyViewController: UIViewController, MessageBannerViewCon
 
     self.tableView.tableHeaderView = headerContainerView
     self.headerView.widthAnchor.constraint(equalTo: self.tableView.widthAnchor).isActive = true
-    self.tableView.ksr_sizeHeaderFooterViewsToFit()
 
     self.viewModel.inputs.viewDidLoad()
   }
 
+  override func viewDidLayoutSubviews() {
```

Comment:
I think one of the reasons we don't really need to force the header/footer to resize in `viewDidLoad` is that at that time the superview still doesn't have proper dimensions...so I think the next best place is `viewDidLayoutSubviews` which should be called when the `bounds` change (final layout to screen bounds (parent bounds) OR rotation)